integr8ly/tutorial-web-app
# Create AMQ service

@david-martin Mind taking a look.

* Provision the AMQ ServiceInstance
* Watch the AMQ Statefulset and create a secret from its credentials once it's ready
* Create an annotation on the AMQ ServiceInstance with the service route

I need to add some docs to this stuff too.

## Verification

* Run `OPENSHIFT_HOST=<your-cluster> yarn start:dev`
* Login
* Ensure: AMQ is provisioned. A secret named amq-broker-credentials is created. You can access the AMQ dashboard from the webapp.
